x-pack/plugins/cases/common/api/cases/constants.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * The field used for authorization in various entities within cases.
+ */
+export const OWNER_FIELD = 'owner';
```

Comment:
Using this so we don't have to use the `owner` string a number of different places in the types and when constructing saved object filters.

Reply:
Out of curiosity, why this field. I feel like you could technically do this for `id`, `version`, and really any other common api field. Doesn't hurt anything, just not sure if it's necessary

Reply:
Yeah true, I guess I was thinking this might be helpful for the places where we're constructing a filter string to avoid accidental spelling errors in the future 🤷‍♂️  it's probably not necessary.